library/core/src/cmp.rs

```diff
@@ -1384,7 +1384,8 @@ mod impls {
     macro_rules! ord_impl {
         ($($t:ty)*) => ($(
             #[stable(feature = "rust1", since = "1.0.0")]
-            impl PartialOrd for $t {
+            #[rustc_const_unstable(feature = "const_primitive_ords", issue = "none")]
+            impl const PartialOrd for $t {
```

Comment:
All changes in this file redundant to #92390.

Reply:
Let's wait for that to land first too, then.